### ISSUE TITLE
How do you style notifications in the stack?

### ISSUE THREAD

**Issue by kwhitaker** [pburtchaell/react-notification#48]
I see some code in the `NotifcationStack` component, but when I try to provide a `style` key to my messages, it doesn't seem to be recognized.

```
errors.push({
      message: 'FOOBAR',
      key: 1,
      style: {background: 'red'},
    })
return <NotificationStack notifications={errors} onDismiss={() => {}} />
```

**Comment by pburtchaell** [comment]
I think there are some overarching issues with styles in general for the `NotifcationStack`—see issue #47. I am not available presently, but when I have some time this week I'm going to review the code and make sure this issue is resolved. :smile:

**Comment by pburtchaell** [comment]
This should be fixed in `react-notification@5.0.0`. Keep in mind the [breaking changes](https://github.com/pburtchaell/react-notification/releases/tag/5.0.0), which are documented in the release as well as the README.
